package.json

```diff
@@ -5,7 +5,12 @@
   "main": "index.js",
   "scripts": {
     "test": "mocha --require @babel/register './src/**/**test.js'",
-    "start": "nodemon --exec babel-node src/index.js"
+    "start-dev": "nodemon --exec babel-node src/index.js",
+    "start": "node dist/index.js",
+    "babel-build": "babel src -d dist",
+    "clean": "rm -rf dist && mkdir dist",
+    "build": "npm run clean && npm run babel-build",
+    "heroku-postbuild": "npm run build"
   },
   "author": "Andela Simulations Programme",
   "license": "MIT",
```

Comment:
Nice one here, but I feel you should add an info to the description of tasks to show that scripts were added to `package.json`. This will ensure that we are more precise about the implementation steps that was carried out.

Reply:
Thank you @Ileriayo, If you check the  " What does this PR do?"  section in the body of the PR, you will see that I indicated that npm scripts were added, the npm scripts being what you indicated.